packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -755,6 +756,19 @@ void main() {
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS }));
 
   group('ClipboardStatusNotifier', () {
+    setUp(() {
+      // TODO(justinmc): Currently on Android and iOS, ClipboardStatus.pasteable
+      //                 is always returned. Once both platforms properly use
+      //                 `hasStrings` to check whether the clipboard has any
+      //                 content, try to see if this override can be removed.
+      //                  https://github.com/flutter/flutter/issues/74139
+      debugDefaultTargetPlatformOverride = TargetPlatform.linux;
```

Comment:
Instead of manually setting debugDefaultTargetPlatformOverride, could you use the `variant` parameter to specify that the tests run on only the platforms that work?  Here's an example: https://github.com/flutter/flutter/blob/8034286b5f90eb3e8006b7b938880ecae25b1695/packages/flutter/test/material/text_field_test.dart#L271